### ISSUE TITLE
Factory returns EntityFactory, which is not assignable

### ISSUE THREAD

**Issue by 0b10** [w3tecch/typeorm-seeding#30]
I can't understand how relations work for entity factories. Below is the snippet of code from the [README](https://github.com/w3tecch/typeorm-seeding#using-model-factories) (minus some things for readability)

```ts
import Faker from 'faker';
import { define } from 'typeorm-seeding';
import { Pet } from '../entities'

define(Pet, (faker: typeof Faker, settings: undefined) => {
  // ...other stuff
  pet.user = factory(User)() // invalid type
  return pet
})
```

`factory(User)()` returns an `EntityFactory<User, undefined>` object, which is not a `User` type, and thus not assignable to `pet.user`.

Am I just doing it wrong?

**Comment by astroash** [comment]
Any update on this issue? Or workarounds for this?

**Comment by hirsch88** [comment]
hi @0b10  and @astroash 
I have finally included this feature! check out version 1.4.0

Just use any type ;-)

```TypeScript
define(Pet, (faker: typeof Faker, settings: undefined) => {
  // ...other stuff
  pet.user = factory(User)() as any
  return pet
})
```
